circuits/circom/prover/wasm/generate_witness.js

```diff
@@ -0,0 +1,22 @@
+const wc = require('./witness_calculator.js')
+const { readFileSync, writeFile } = require('fs')
+
```

Comment:
when processing command line args with node js, you may consider using [`yargs`](https://yargs.js.org/)
Maybe overkill here, but it is helpful to build nice menus and usage prompts

Reply:
Ahhh, so everything in `wasm` is actually autogenerated by circom! I have moved it from `wasm` to `generated` to make that clear. Perhaps it would have been even clearer to keep it out of git at all, but I think it's nice to have it pre built.

Sorry for making you read through that - we could post your critiques upstream to circom!

Reply:
ooh you mean this file is generated by circom?